### ISSUE TITLE
Missing 'Last state change' from Flapjack::Diner.status_report_checks

### ISSUE THREAD

**Issue by Sarah-E-Greene** [flapjack/flapjack#678]
In the flapjack UI, you can get the last state change of a check.

This isn't reflected in the API, when using Flapjack::Diner.status_report_checks:

Flapjack::Diner.status_report_checks(check_id)
=> [{:name=>"HOST",
  :state=>"ok",
  :enabled=>true,
  :summary=>"PING OK - Packet loss = 0%, RTA = 0.25 ms",
  :details=>"",
  :perfdata=>nil,
  :in_unscheduled_maintenance=>false,
  :in_scheduled_maintenance=>false,
  :last_update=>1398775600,
  :last_problem_notification=>1364367519,
  :last_recovery_notification=>1364367740,
  :last_acknowledgement_notification=>nil,
  :links=>
   {:entity=>["523e8c70-7d81-4d8c-8e16-a8832c32ccce"],
    :check=>["google.com:HOST"]}}]

A use case I have for this would be "I've just applied system updates, and I want to see if anything has started newly alerting".
